<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TMainWnd</class>
 <widget class="QMainWindow" name="TMainWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1143</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Store</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: #F5FFFA;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="categoryLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QTableView" name="categoryTable">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QHeaderView {
    background-color: #fff;
    font-size:13px;
}
QHeaderView::section:horizontal {
    color: #fff;
    border-style: solid;
    background-color: qlineargradient( x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #696969, stop: 1 #A9A9A9);
 }
QTableView {
    border: 2px solid #F5FFFA;
    border-top-color: #696969;
    border-radius: 4px;
    background-color: #F5FFFA;
    background-image: url(:/img/other/background.png);
    gridline-color: #777;
    selection-background-color: #778899;
    color:#333;
    font-size:20px;
	text-align: center;
 }</string>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>100</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>100</number>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <layout class="QVBoxLayout" name="productLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QTableView" name="productTable">
          <property name="font">
           <font>
            <family>Verdana</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QHeaderView {
    background-color: #fff;
    font-size:13px;
}
QHeaderView::section:horizontal {
    color: #fff;
    border-style: solid;
    background-color: qlineargradient( x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #696969, stop: 1 #A9A9A9);
 }
QTableView {
    border: 2px solid #F5FFFA;
    border-top-color: #696969;
    border-radius: 4px;
    background-color: #F5FFFA;
    background-image: url(:/img/other/background.png);
    gridline-color: #777;
    selection-background-color: #778899;
    color:#333;
    font-size:12px;
 }</string>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>20</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>20</number>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="controlFrame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>36</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="buttonLayout" stretch="0,0,0,0">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="appendBtn">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>28</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: #7FFFD4;
    border-width: 2px;
    border-radius: 8px;
}</string>
         </property>
         <property name="text">
          <string>Append product</string>
         </property>
         <property name="icon">
          <iconset resource="res/store.qrc">
           <normaloff>:/images/plus</normaloff>:/images/plus</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeBtn">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>28</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: #FFB6C1;
    border-width: 2px;
    border-radius: 8px;
}</string>
         </property>
         <property name="text">
          <string>Remove product</string>
         </property>
         <property name="icon">
          <iconset resource="res/store.qrc">
           <normaloff>:/images/minus</normaloff>:/images/minus</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="searchBtn">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>28</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: #E0FFFF;
    border-width: 2px;
    border-radius: 8px;
}</string>
         </property>
         <property name="text">
          <string>Search product</string>
         </property>
         <property name="icon">
          <iconset resource="res/store.qrc">
           <normaloff>:/images/search</normaloff>:/images/search</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exportBtn">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>28</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: #98FB98;
    border-width: 2px;
    border-radius: 8px;
}</string>
         </property>
         <property name="text">
          <string>Export to CSV</string>
         </property>
         <property name="icon">
          <iconset resource="res/store.qrc">
           <normaloff>:/images/export_csv</normaloff>:/images/export_csv</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="res/store.qrc"/>
 </resources>
 <connections/>
</ui>
